fem/gslib.cpp

```diff
@@ -150,6 +157,30 @@ void FindPointsGSLIB::FindPoints(const Vector &point_pos,
    }
 }
 
+void FindPointsGSLIB::FindPoints(const Vector &point_pos)
+{
+   const int points_cnt = point_pos.Size() / dim;
+   gsl_code.SetSize(points_cnt);
+   gsl_proc.SetSize(points_cnt);
+   gsl_elem.SetSize(points_cnt);
+   gsl_ref.SetSize(points_cnt * dim);
+   gsl_dist.SetSize(points_cnt);
+
+   FindPoints(point_pos, gsl_code, gsl_proc, gsl_elem, gsl_ref, gsl_dist);
+}
+
+void FindPointsGSLIB::FindPoints(Mesh &m, const Vector &point_pos)
+{
+   if (!setupflag || (mesh != &m) )
+   {
+      const double rel_bbox_el = 0.05;
+      const double newton_tol  = 1.0e-12;
+      const int npts_at_once   = 256;
```

Comment:
Is it desired that the user can control these parameters from the outside? If yes, these parameters could be added as arguments to this method with prescribed default values.